code/game/machinery/iv_drip.dm

```diff
@@ -244,11 +285,11 @@
 ///Called when an iv is detached. doesnt include chat stuff because there's multiple options and its better handled by the caller
 /obj/machinery/iv_drip/proc/detach_iv()
 	SEND_SIGNAL(src, COMSIG_IV_DETACH, attached)
-
+	transfer_rate = MIN_IV_TRANSFER_RATE
 	attached = null
 	update_appearance()
 
-/obj/machinery/iv_drip/proc/get_reagent_holder()
+/obj/machinery/iv_drip/proc/get_reagents()
```

Comment:
Autodoc this (I know it seems obvious but I already got confused by the difference between reagents_holder and reagents)